include/lsst/afw/geom/SpanSet.h

```diff
@@ -617,6 +620,23 @@ class SpanSet {
      */
     std::vector<std::shared_ptr<geom::SpanSet>> split() const;
 
+protected:
```

Comment:
As we are not expecting anything to inherit from SpanSet, this functionality was moved to private.